src/DIRAC/FrameworkSystem/Service/ProxyManagerHandler.py

```diff
@@ -123,13 +119,13 @@ def export_completeDelegationUpload(self, requestId, pemChain):
         :return: S_OK(dict)/S_ERROR() -- dict contain proxies
         """
         credDict = self.getRemoteCredentials()
-        userId = "%s:%s" % (credDict["username"], credDict["group"])
+        userId = f'{credDict["username"]}:{credDict["group"]}'
         retVal = self.__proxyDB.completeDelegation(requestId, credDict["DN"], pemChain)
         if not retVal["OK"]:
-            gLogger.error("Upload proxy failed", "id: %s user: %s message: %s" % (requestId, userId, retVal["Message"]))
-            return self.__addKnownUserProxiesInfo(retVal)
-        gLogger.info("Upload %s by %s completed" % (requestId, userId))
-        return self.__addKnownUserProxiesInfo(S_OK())
+            gLogger.error("Upload proxy failed", f"id: {requestId} user: {userId} message: {retVal['Message']}")
+            return retVal
+        gLogger.info(f"Upload {requestId} by {userId} completed")
+        return S_OK(self.__generateUserProxiesInfo())
```

Comment:
For my info: why changing from `return self.__addKnownUserProxiesInfo(S_OK())` to `return S_OK(self.__generateUserProxiesInfo())`?

Reply:
Old logic create:
```python
{
  "OK": True,
  "Value": None,
  "proxies": {"/SomeDN..": "some data.."}
}
```
This changes:
```python
{
  "OK": True,
  "Value": {"/SomeDN..": "some data.."}
}
```

Client understand both:
```python
    def uploadProxy(self, proxy=None, restrictLifeTime=0, rfcIfPossible=False):
        ...
        # Upload!
        result = rpcClient.completeDelegationUpload(reqDict["id"], retVal["Value"])
        if not result["OK"]:
            return result
        return S_OK(result.get("proxies") or result["Value"])